apps/trainer/main.py

```diff
@@ -0,0 +1,237 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Usage: python -m apps.trainer.main --config apps/grpo/qwen3_32b.yaml
```

Comment:
nit, can we re-name this to rl_trainer? The reason is because we have sft, sft_v2 and then if we have a separate trainer it may confuse even further lol